cajita/src/Cajita_LocalGrid.hpp

```diff
@@ -55,6 +55,9 @@ class LocalGrid
     //! \brief Get the global grid that owns the local grid.
     const GlobalGrid<MeshType>& globalGrid() const;
 
+    //! \brief Get a mutable version of the global grid that own the local grid
+    GlobalGrid<MeshType>& mutGlobalGrid();
```

Comment:
you can call this function `globalGrid()`. Its OK to have a const and non-const version of the same accessor.

Reply:
Ok, done.